packages/contracts/src/interfaces/IRelayHub.sol

```diff
@@ -22,6 +22,14 @@ interface IRelayHub {
         uint256 minimumUnstakeDelay;
         // Minimum stake a relay can have. An attack on the network will never cost less than half this value.
         uint256 minimumStake;
+        // Developers information
+        DevInfo devInfo;
+    }
+
+    struct DevInfo {
+        address addr;
+        // 0 < fee < 100, as percentage of total charge from paymaster to relayer
+        uint256 fee;
```

Comment:
1. I dislike inner struct too.
2. name it "pct", since its percent, like normal fee.

Reply:
1. Ok ok, I will flatten it.
2. Ok